.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -259,7 +259,7 @@ stages:
         build: true
         target: builder
         baseImage: $(baseImage)
-        command: "Clean BuildTracerHome ZipTracerHome"
+        command: "Clean BuildTracerHome BuildNativeLoader BuildProfilerHome ZipTracerHome"
```

Comment:
Done
I move the profiler compilation to a separate job (so  it can be run in parallel)
During the `build_profiler_linux` job, I also build the native tests and upload the output directory. If I do it later (copying the working directory), it will recompile the profiler and its dependencies :/ ) Compiling the native tests is not that much.
Now, I just download the output directory, and run the tests binary(-es).